src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -1610,19 +1794,21 @@ private void hitBlockWithToolInHand(@Nullable final BlockPos blockPos, final boo
             if (!world.isRemote)
             {
                 MineColonies.getNetwork().sendToAllAround(
-                  //todo: correct side
-                  new BlockParticleEffectMessage(blockPos, world.getBlockState(blockPos), 1),
-                  new NetworkRegistry.TargetPoint(world.provider.getDimension(), blockPos.getX(), blockPos.getY(), blockPos.getZ(), BLOCK_BREAK_PARTICLE_RANGE));
+                        //todo: correct side
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)